msmbuilder/tests/test_featurizer.py

```diff
@@ -48,6 +50,30 @@ def test_SubsetAtomPairs2():
     any([eq(x, x0) for (x, x0) in zip(X_all, X_all0)])
 
 
+def test_function_featurizer():
+    dataset = fetch_alanine_dipeptide()
+    trajectories = dataset["trajectories"]
+    trj0 = trajectories[0][0]
+    #use the dihedral to compute phi for ala
+
+
+    atom_ind = [[4, 6, 8,14]]
+    func = compute_dihedrals
+    f = FunctionFeaturizer(func, indices=atom_ind)
+    res1 = f.transform([trj0])
+
+    def funcception(trj):
+        return compute_phi(trj)[1]
+
+    f = FunctionFeaturizer(funcception)
+    res2 = f.transform([trj0])
+
+    f3 = DihedralFeaturizer(['phi'], sincos=False)
+    res3 = f3.transform([trj0])
+
+    assert res1==res2==res3
```

Comment:
Does this work? Shouldn't you use `np.testing.assert_array_equal` or something like that to check elementwise equality